client/src/components/profile/components/time-line.tsx

```diff
@@ -122,7 +122,7 @@ function TimelineInner({
     const { challengeTitle, challengePath, certPath } = idToNameMap.get(id);
     return (
       <tr className='timeline-row' key={id}>
-        <td>
+        <td className='solutions-dropdown'>
```

Comment:
This data cell is for displaying the challenge name, not for the view button/dropdown. The view button/dropdown is the next `td` tag below but instead of adding the class to the `td` tag it's more consistent to update the code that renders the dropdown button as already one [function](https://github.com/freeCodeCamp/freeCodeCamp/blob/f98db0e1ac8c51365d2556ef53b7c0f2740405c9/client/src/components/solution-display-widget/index.tsx#L97-L122) does it. Similarly `ShowMultifileProjectSolution` function should also be updated.

Reply:
That's make sense, I will start working on it

Reply:
I have nested `td` in `.solutions-dropdown` 

```html
<div className='solutions-dropdown'>
  <td>
   ....
  </td>
</div>
```

and the same to `ShowMultifileProjectSolution`

```html
<div className='solutions-dropdown'>
      <DropdownButton
        block={true}
        bsStyle='primary'
        className='btn-invert'
        id={`dropdown-for-${id}`}
        title={t('buttons.view')}
      >
        <MenuItem bsStyle='primary' onClick={showUserCode}>
          {t('buttons.show-code')}
        </MenuItem>
        <MenuItem bsStyle='primary' onClick={showProjectPreview}>
          {t('buttons.show-project')}
        </MenuItem>
      </DropdownButton>
    </div>
```